Add pipeline for test deployment

fureigh opened the pull request:
Closely modeled after https://github.com/18F/cf-hello-worlds/blob/master/pipeline.yml.

Closes https://github.com/18F/cf-ex-drupal/issues/6.

fureigh committed:
Add pipeline for test deployment

Closes https://github.com/18F/cf-ex-drupal/issues/6.

jmcarp commented on pipeline.yml:
```diff
@@ -0,0 +1,50 @@
+jobs:
+- name: deploy-drupal-7
+  plan:
+  - aggregate: *plan-gets
```
This yaml reference isn't defined.

jmcarp commented on pipeline.yml:
```diff
@@ -0,0 +1,50 @@
+  on_failure:
+    put: slack
+    params:
+      <<: *slack-params
```
Same as above--this reference need to be defined for this to be valid yaml. Also, I used references in the sample apps pipeline to avoid duplicating boilerplate config, but since this pipeline only deploys one app, it would probably be clearer not to use them at all.

jmcarp commented:
Unless somebody manually creates a database service with the expected name in the space where this is going to run, it looks like this won't work. What do you think about adding a job to ensure the database exists?

fureigh commented:
Thanks for the feedback, @jmcarp! 👍 to adding a job to ensure the database exists... or, better yet, using the manifest file to do that.

jmcarp commented:
Sounds good! I'm not sure which manifest file you mean--cloud foundry manifests can bind services to applications but not create services, as far as I know. Anyway, let me know if you want to pair on this sometime. This is a good start, and I'm excited to see it working!

fureigh committed:
Merge branch 'master' of github.com:18F/cf-ex-drupal

fureigh committed:
Update dbname to be more specific

https://github.com/18F/cf-ex-drupal/issues/6

fureigh committed:
Define references and add service

This commit, hewing more closely to the example of
https://github.com/18F/cf-cdn-service-broker/tree/master/ci, seeks
to address @jmcarp's feedback. Hat tip to @jcscottiii for pointing
me to that repo.

https://github.com/18F/cf-ex-drupal/issues/6

fureigh commented:
@jmcarp Thanks for the feedback! As you can see, I'm new to Concourse.

Returning to this as part of the IP sprint, I spent some more time with concourse.ci and https://github.com/18F/cf-cdn-service-broker/. Ready for another look.
